tests/test_pe_reports.py

```diff
@@ -1,33 +1,79 @@
 """File to be used to create test for pe-reports."""
 
 # Standard Python Libraries
-import unittest
+import logging
+import sys
+from unittest.mock import patch
+
+# Third-Party Libraries
+import pytest
+
+# cisagov Libraries
+import pe_reports.report_generator
+
+log_levels = (
+    "debug",
+    "info",
+    "warning",
+    "error",
+    "critical",
+)
```

Comment:
Is `log_levels` used anywhere in this file?  If not then it should be removed.

It would be nice to see the log level verified in `test_log_levels()` but that may be a change that's better to make to the skeleton itself.  What say you, @mcdonnnj?

Reply:
It should see usage with
https://github.com/cisagov/pe-reports/blob/10a9aff84e6d2ec21e33fc190524cfc3f6798f2f/tests/test_pe_reports.py#L57
which is then used in
https://github.com/cisagov/pe-reports/blob/10a9aff84e6d2ec21e33fc190524cfc3f6798f2f/tests/test_pe_reports.py#L68

Reply:
OK, thanks for pointing out that `log_levels` is actually used.  My ancient eyes don't see like they used to!

I still don't see where in `test_log_levels()` we actually verify that the log level has been set.

Reply:
It calls `pe_reports.report_generator.main()` with each (valid) log level and tests [whether the root logger has a handler](https://github.com/cisagov/pe-reports/blob/4d64c0f442b65f47d799e7809bd4a68fa395dcb6/tests/test_pe_reports.py#L75-L78).  This test is virtually unchanged from [our skeleton](https://github.com/cisagov/skeleton-python-library/blob/4ef2364701c0db074c423f740062b2d7c197bea8/tests/test_example.py#L74-L86).

Reply:
Right, I guess I am saying that the skeleton test needs some additional content.  It doesn't actually check that the correct log level was assigned.  If (being specific to this PR) the function `pe_reports.report_generator.main()` just set the logging level to `DEBUG` no matter what you passed in on the command line, then `test_log_levels()` would still pass.

I will create an issue in [cisagov/skeleton-python-library](https://github.com/cisagov/skeleton-python-library) and we can move the discussion there.

Reply:
Please continue this discussion at cisagov/skeleton-python-library#79.

Reply:
I understand what you are saying now and agree that would be a good test to add.